pkg/cvo/status.go

```diff
@@ -347,18 +352,34 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 // unavailable AND the general payload task making progress towards its goal. An operator is given 40 minutes since
 // its last update to go ready, or an hour has elapsed since the update began, before the condition is ignored.
 func convertErrorToProgressing(history []configv1.UpdateHistory, now time.Time, status *SyncWorkerStatus) (reason string, message string, ok bool) {
-	if len(history) == 0 || status.Failure == nil || status.Reconciling || status.LastProgress.IsZero() {
+	if len(history) == 0 || status.Failure == nil || status.Reconciling {
 		return "", "", false
 	}
-	if now.Sub(status.LastProgress) > 40*time.Minute || now.Sub(history[0].StartedTime.Time) > time.Hour {
+	if now.Sub(history[0].StartedTime.Time) > time.Hour {
```

Comment:
I think we can drop this "has it been an hour since we started updating?" condition too.  "We are updating, and it's been more than an hour" is not in itself enough of a reason to go `Failing=True`.  The hour bit landed in c2ac20fa172da (#158), along with the Godoc comment "or an hour has elapsed since the update began", which I want to drop along with this block.

Reply:
redone